components/react/modal.tsx

```
@@ -191,16 +440,17 @@
             address={current?.address}
           />
         );
+
       case ModalView.Connecting:
+        // Decide a tailored message if it's a WalletConnect flow
         let subtitle: string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)